examples/fluids/src/differential_filter.c

```diff
@@ -64,9 +62,8 @@ PetscErrorCode DifferentialFilterCreateOperators(Ceed ceed, User user, CeedData
       char                field_name[PETSC_MAX_PATH_LEN];
       CeedElemRestriction elem_restr_filter;
       CeedBasis           basis_filter;
-
-      PetscCall(GetRestrictionForDomain(ceed, dm_filter, 0, 0, 0, i, num_qpts_1d, 0, &elem_restr_filter, NULL, NULL));
-      CeedBasisCreateTensorH1Lagrange(ceed, dim, diff_filter->num_field_components[i], num_nodes_1d, num_qpts_1d, CEED_GAUSS, &basis_filter);
+      PetscCall(GetRestrictionForDomain(ceed, dm_filter, 0, 0, 0, i, -1, 0, &elem_restr_filter, NULL, NULL));
```

Comment:
Silly question, but why `-1` in the args?

Reply:
Mostly because I can't put a `NULL` there. That argument isn't used unless it's generating a collocated element restriction. Conversely the (often) four `0` zero arguments before it _are_ always used so I put -1 there to make it look different.

Should probably just create a separate "GetCollocatedRestriction" function, but I was trying to tackle one thing at a time.

Reply:
Note - places where you aren't setting up a strided restriction you should probably be calling `CreateRestrictionFromPlex` directly. Then you wouldn't have to worry about this `-1` business

Reply:
TBD off of #1267